canonical-websites/snapcraft.io
# Attach a GA event to snap Install button

As discussed during the GA mentoring session last month, this fires a GA event when the install button is triggered on snap details pages.

## QA

1. Open a snap page such as  http://snapcraft.io-pr-238.run.demo.haus/scummvm/
2. Click on the **Install** button
3. Ensure the event has been properly logged by going to the real-time/events view of the "0) Device - DevSites" GA property
4. Click on the most recent "Snap details" link in the "Event Category" col, ensure the "Event label" col contains the name of the snap you tried to install
